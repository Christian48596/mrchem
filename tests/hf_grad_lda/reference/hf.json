{
  "input": {
    "constants": {
      "angstrom2bohrs": 1.8897261246257702,
      "dipmom_au2debye": 2.5417464739297717,
      "electron_g_factor": -2.00231930436256,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "geometric_derivative": {
          "geom-1": {
            "operator": "h_nuc_grad",
            "precision": 0.001,
            "smoothing": 0.001
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655945
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.719760770272972,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.8969784618809413,
            -0.28413890133310815,
            1.4395387562135316
          ],
          "vector_el": [
            22.58536563229331,
            4.622934568926525,
            -6.451459662288635
          ],
          "vector_nuc": [
            -23.482344094174252,
            -4.9070734702596335,
            7.890998418502167
          ]
        }
      },
      "geometric_derivative": {
        "geom-1": {
          "electronic": [
            0.1915036906118746,
            -2.076842414093538,
            -0.30859536452891734,
            -1.7451176662202608,
            -0.5501106508673969,
            2.8002295486765068
          ],
          "electronic_norm": 3.9540513524659002,
          "nuclear": [
            -1.5689195606619417,
            -0.49421531749272585,
            2.5173387449218514,
            1.5689195606619417,
            0.49421531749272585,
            -2.5173387449218514
          ],
          "nuclear_norm": 4.252705427032731,
          "total": [
            -1.3774158700500672,
            -2.571057731586264,
            2.208743380392934,
            -0.17619810555831905,
            -0.05589533337467101,
            0.28289080375465536
          ],
          "total_norm": 3.6742831853559217
        }
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -15.92379519939184,
          3.23708012750867,
          4.7545830018739474,
          4.7770989185484645,
          4.777662945436716
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": 3.245259587951912
      },
      "scf_energy": {
        "E_ee": 94.64341476549207,
        "E_eext": 0.0,
        "E_el": -86.48817471847417,
        "E_en": -329.47163126955746,
        "E_kin": 163.65718962808455,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -81.28586257745523,
        "E_x": 0.0,
        "E_xc": -15.317147842493323,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.64341476549207,
        "E_eext": 0.0,
        "E_el": -86.48817471847417,
        "E_en": -329.47163126955746,
        "E_kin": 163.65718962808455,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -81.28586257745523,
        "E_x": 0.0,
        "E_xc": -15.317147842493323,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
