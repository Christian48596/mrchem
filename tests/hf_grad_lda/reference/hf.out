                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         vecmult                                  ***
***         Git commit hash    7f93542d94af5a58c655-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Tue Feb 28 14:33:44 2023 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.5.0-alpha
 Git branch            : master
 Git commit hash       : 134cf947b28cb88ce9d9-dirty
 Git commit author     : Stig Rune Jensen
 Git commit date       : Wed Nov 23 14:47:03 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.996812789590e-01
    1    2    p          :                              9.998694293756e-01
    2    2    p          :                              9.999998231017e-01
    3    2    p          :                              9.999999836389e-01
    4    2    p          :                              9.999998001090e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Relativity              : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.657189628085
 E-N energy              :            (au)               -329.471631269557
 Coulomb energy          :            (au)                 94.643414765492
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.317147842493
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -86.488174718474
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.128586257746e+01
                         :      (kcal/mol)             -5.100764887474e+04
                         :        (kJ/mol)             -2.134160028919e+05
                         :            (eV)             -2.211901002933e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.923795199392
    1    2    p          :            (au)                  3.237080127509
    2    2    p          :            (au)                  4.754583001874
    3    2    p          :            (au)                  4.777098918548
    4    2    p          :            (au)                  4.777662945437
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  3.245259587952
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                163.657189628085
 E-N energy              :            (au)               -329.471631269557
 Coulomb energy          :            (au)                 94.643414765492
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.317147842493
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -86.488174718474
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.128586257746e+01
                         :      (kcal/mol)             -5.100764887474e+04
                         :        (kJ/mol)             -2.134160028919e+05
                         :            (eV)             -2.211901002933e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.923795199392
    1    2    p          :            (au)                  3.237080127509
    2    2    p          :            (au)                  4.754583001874
    3    2    p          :            (au)                  4.777098918548
    4    2    p          :            (au)                  4.777662945437
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  3.245259587952
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       22.585366        4.622935       -6.451460
 Magnitude               :            (au)                       23.939332
                         :         (Debye)                       60.847712
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189426
---------------------------------------------------------------------------
 Total vector            :       -0.896978       -0.284139        1.439539
 Magnitude               :            (au)                        1.719761
                         :         (Debye)                        4.371196
===========================================================================


===========================================================================
                       Geometric Derivative (geom-1)
---------------------------------------------------------------------------
 Total                   :       -1.377416       -2.571058        2.208743
                         :       -0.176198       -0.055895        0.282891
 Norm                    :            (au)                        3.674283
---------------------------------------------------------------------------
 Nuclear                 :       -1.568920       -0.494215        2.517339
                         :        1.568920        0.494215       -2.517339
 Norm                    :            (au)                        4.252705
---------------------------------------------------------------------------
 Electronic              :        0.191504       -2.076842       -0.308595
                         :       -1.745118       -0.550111        2.800230
 Norm                    :            (au)                        3.954051
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
